robots/configBasedRobot.py

```diff
@@ -21,6 +22,9 @@ def __init__(self, period: float = 0.02) -> None:
 
         self.configMapper = utils.configMapper.ConfigMapper(config, configPath)
 
+        self.xboxController = wpilib.XboxController(0)
```

Comment:
Love this! In the future, would love to refactor the `getStick()` function to take an already instantiated controller instead of automatically instantiating at port 0